pay-api/src/pay_api/services/payment_account.py

```diff
@@ -675,15 +676,21 @@ def deduct_eft_credit(cls, invoice: InvoiceModel):
         # Deduct credits and apply to the invoice
         now = datetime.now()
         for eft_credit in eft_credits:
+            credit_invoice_link = EFTCreditInvoiceLinkModel(
+                eft_credit_id=eft_credit.id,
+                invoice_id=invoice.id
+            )
+            _ = credit_invoice_link.save() if auto_save else db.session.add(credit_invoice_link)
```

Comment:
helper function, probably cleaner than doing this

Reply:
I was thinking the same thing, will add it to the base so we can re-use this easily